providers/README.md

```diff
@@ -0,0 +1,10 @@
+# cloud-provider-gcp/providers
+
+Currently feature development is no longer accepted in `k8s.io/kubernetes/staging/src/k8s.io/legacy-cloud-providers/<provider>`. 
+This directory is to support further development for in-tree cloud provider gce.
+
+`cloud-provider-gcp/providers/gce` will contain the files moved from `k8s.io/kubernetes/staging/src/k8s.io/legacy-cloud-providers/gce`. And `cloud-provider-gcp/providers` will
+be released separately.
+Later, k8s.io/kubernetes will switch the dependency from `k8s.io/legacy-cloud-providers/gce` to `cloud-provider-gcp/providers`. `k8s.io/legacy-cloud-providers/gce` will be deleted.
+
+After the dependency switch completed, the feature development could continue in `cloud-provider-gcp/providers`.
```

Comment:
"Could" suggests we are seriously considering an alternative and trying to warn people there is a likelihood feature development will move to somewhere else. If that is true, fine. Otherwise I would suggest using "will".

Reply:
Updated. Thank you